packages/cli/src/util/terragrunt/tasks/deployModuleTask.ts

```diff
@@ -141,6 +144,14 @@ export async function buildDeployModuleTask<T extends {}>(inputs: {
         title: context.logger.applyColors(`${taskTitle} ${module}`, {
             lowlights: [module],
         }),
+        skip: async () => {
```

Comment:
I don't like this being in this module like this as it will break other uses of this task.

Make it `skipIfAlreadyApplied` and set the default to `false`. If `false`, nothing new occurs. If `true`, then the module should be skipped if `pfData.status === "applied"`.